src/components/Blogs/Blogs.tsx

```diff
@@ -0,0 +1,61 @@
+import { Flex, Text } from '@chakra-ui/react';
+import { useEffect, useState } from 'react';
+import { useNavigate } from 'react-router';
+
+import { ROUTES } from '~/constants/routes';
+import { useGetBloggersQuery } from '~/query/services/bloggers-api/bloggers-api';
+import { getUserIdFromToken } from '~/utils/getUserIdFromToken';
+
+import { NewRecipesSection } from '../Main/NewRecipesSection/NewRecipesSection';
+import { Bloggers } from './Bloggers/Bloggers';
+import { Favorites } from './Favorites/Favorites';
+
+export const Blogs = () => {
+    const navigate = useNavigate();
+    const userId = getUserIdFromToken();
+    const [limit, setLimit] = useState<string>('9');
+
+    const { data, refetch, isError } = useGetBloggersQuery({ currentUserId: userId, limit });
```

Comment:
```suggestion
    const { data, refetch, isError } = useGetBloggersQuery({ currentUserId: userId, limit: String(limit) });
```
А ещё лучше приводить к строке сразу в апишке

и не придётся играть в числа-строки (как на 16 строке)